resources/views/errors/generic.blade.php

```diff
@@ -1 +1,7 @@
-{{ $status_code }} {{ $status_message }}
+@extends('layouts.base')
+
+@section('content')
+    <h1>{{ $status_code }}</h1>
```

Comment:
There's a reason we didn't use the `layouts.base` for this generic error page. I remember it had something to do with API requests also hitting this "stylized" error page. Until that problem is fixed, this modification won't help API consumers.